corehq/apps/users/views/__init__.py

```diff
@@ -1146,8 +1146,6 @@ def post(self, request, *args, **kwargs):
                 data["invited_by"] = request.couch_user.user_id
                 data["invited_on"] = datetime.utcnow()
                 data["domain"] = self.domain
-                # Preparation for location_id to replace supply_point
-                data["location_id"] = data.get("supply_point", None)
```

Comment:
I see you added this in PR1, but can you remind me why? Why wasn't the location id / supply point already being added to the invitation being created?

Reply:
Sorry for the late reply. The `supply_point` is already being added to the invitation. 
In PR1, `location` field is added to the `Invitation`. This line was added to also populate the `location` field with the same value (but with the SQLLocation object instead of just the id) that populates `supply_point`. 

With PR1, both `supply_point` and `location` field exists. This PR will remove `supply_point`